Source/WebCore/css/parser/CSSParserFastPaths.cpp

```diff
@@ -901,6 +901,45 @@ static RefPtr<CSSValue> parseSimpleTransform(CSSPropertyID propertyID, StringVie
     return parseSimpleTransformList(string.characters16(), string.length());
 }
 
+static RefPtr<CSSValue> parseDisplay(StringView string)
```

Comment:
I would add a comment in `consumeDisplay` to make sure we add any future single keywords here too.

Reply:
Added.